Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -53,6 +53,18 @@ protected def con : Con G where
 instance Quotient.group : Group (G ⧸ N) :=
   (QuotientGroup.con N).group
 
+/--
+The congruence relation defined by the kernel of a group homomorphism is equal to its kernel
+as a congruence relation.
+-/
+@[to_additive
+"The additive congruence relation defined by the kernel of an additive group homomorphism is
+equal to its kernel as an additive congruence relation."]
+theorem con_ker_eq_conKer {M : Type*} [Monoid M] (f : G →* M) :
```

Comment:
Could you check the additivized name here is correct? The last part should be `addConKer`.